weetweet.py

```diff
@@ -333,25 +355,45 @@ def stream_message(buffer,tweet):
                 arrow_col = weechat.color('chat_prefix_suffix')
                 reset_col = weechat.color('reset')
                 dict_id = dict_tweet(tweet['target_object']['id_str'])
-                extra_str = "'s tweet " + arrow_col +  "<" + reset_col + dict_id + arrow_col + "> " + reset_col
+                extra_str = "'s tweet " + arrow_col +  "<" + reset_col + dict_id + arrow_col + "> "\
+                        + reset_col
 
         #TODO make the event printing better
-        weechat.prnt_date_tags(buffer, 0, "no_highlight", "%s%s" % (weechat.prefix("network"),
-            tweet['source']['screen_name'] + " " + event_str + " " + tweet['target']['screen_name'] + extra_str))
+        weechat.prnt_date_tags(
+            buffer, 0, "no_highlight", "%s%s" % (weechat.prefix("network"),
+                                                 tweet['source']['screen_name'] + " " + event_str +\
+                                            " " + tweet['target']['screen_name'] + extra_str))
     elif 'friends' in tweet:
-        #This should be the initital message you get listing friend ids when connecting to the twitter stream
-        weechat.prnt(buffer, "%s%s" % (weechat.prefix("network"), "Connected to twitter streaming API."))
-        #Get new tweets since the last update (we might have missed some if this is after a reconnect)
+        #This should be the initital message you get listing
+        #friend ids when connecting to the twitter stream
+        weechat.prnt(buffer, "%s%s" % (weechat.prefix("network"),\
+                "Connected to twitter streaming API."))
+        #Get new tweets since the last update
+        #(we might have missed some if this is after a reconnect)
         #Get latest tweets from timeline
         my_command_cb("silent", buffer, "new")
+    elif 'user' in tweet:
+        name = tweet['user']['screen_name']
+        if name == script_options['screen_name']:
+            arrow_col = weechat.color('chat_prefix_suffix')
+            reset_col = weechat.color('reset')
+            dict_id = dict_tweet(tweet['id_str'])
+            extra_str = " " + dict_id
+            if 'full_text' in tweet:
+                text = tweet['full_text']
+            else:
+                text = tweet['text']
+
+            weechat.prnt_date_tags(buffer, 0, "notify_message", "%s\t%s" % \
+                    (tweet['user']['screen_name'] + " " + dict_id, text))
```

Comment:
Is this a new kind of message? Shouldn't all normal messages been filtered out in `twitter_stream_cb` and sent to `print_tweet_data`?

When will this trigger?